namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-dev/resources/s3.tf

```diff
@@ -12,7 +12,7 @@ module "risk_profiler_digcat_s3_bucket" {
 }
 
 module "viper_digcat_s3_bucket" {
-  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=bucket_policy"
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=2.0"
```

Comment:
You might want to update this to 3.0 as well, for consistency.

Reply:
done